ext/bin/cucyber-init

```diff
@@ -36,6 +36,11 @@ sed -i -e "s/CUCYBER_WIKI_SECRET/${CUCYBER_WIKI_SECRET}/g" /etc/webhook/hooks.js
 sed -i -e "s/CUCYBER_PRESENTATIONS_SECRET/${CUCYBER_PRESENTATIONS_SECRET}/g" /etc/webhook/hooks.json
 sed -i -e "s/CUCYBER_CLASS_SECRET/${CUCYBER_CLASS_SECRET}/g" /etc/webhook/hooks.json
 
+# generate site
+cd /var/www/cucyber
+
+jekyll build
```

Comment:
This is not the right place for this because this file only runs once on container startup. It should be in `cucyber-dev-update` which gets run when a GitHub hook gets called.

Reply:
Added a new commit for this. Haven't tested it though since I don't have the whole webhook stuff setup locally.

Reply:
Yeah, it is kinda inconvenient to do locally. You really only have to start the container and mimic the GitHub hooks with curl but there is admittedly no instructions on that.

Reply:
I'll give this a test when I get a chance. I guess we should leave this in init (like you did) so that it has an initial build for the nginx server when the container starts up.